empty-plugin/src/main/resources/archetype-resources/pom.xml

```diff
@@ -30,7 +30,7 @@
     <properties>
         <revision>1.0</revision>
         <changelist>-SNAPSHOT</changelist>
-        <jenkins.version>2.303.1</jenkins.version>
+        <jenkins.version>2.303.3</jenkins.version>
```

Comment:
Never really settled https://github.com/jenkinsci/pipeline-maven-plugin/pull/368#issuecomment-918223943 did we? https://www.jenkins.io/doc/developer/plugin-development/choosing-jenkins-baseline/ is still advocating .1. My personal recommendation is .latest like this, especially in this case since .3 contains https://www.jenkins.io/security/advisory/2021-11-04/.

Reply:
there was enforcer errors without it, a pipeline plugin requires .3.